I think after update `signal` feature should be enabled on all platforms, not only `freebsd`.

On macOs and Linux it doesn't build either.


```
error[E0432]: unresolved imports `nix::sys::signal::Signal`, `nix::unistd::Pid`
   --> src/main.rs:13:25
    |
 13 |     sys::signal::{self, Signal},
    |                         ^^^^^^ no `Signal` in `sys::signal`
 14 |     unistd::Pid,
    |     ^^^^^^^^^^^ no `Pid` in `unistd`
    |
    = help: consider importing this type alias instead:
            ctrlc::Signal
note: found an item that was configured out
   --> /Users/bazhenov/Developer/ttr/.sandbox/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/nix-0.29.0/src/unistd.rs:174:12
    |
168 | #![feature = "process"]
    |    ------------------- the item is gated behind the `process` feature
...
174 | pub struct Pid(pid_t);
    |            ^^^

error[E0425]: cannot find function `kill` in module `signal`
    --> src/main.rs:168:25
     |
 168 |                 signal::kill(Pid::from_raw(pid), Signal::SIGINT).unwrap()
     |                         ^^^^ not found in `signal`
     |
note: found an item that was configured out
    --> /Users/bazhenov/Developer/ttr/.sandbox/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/nix-0.29.0/src/sys/signal.rs:1061:8
     |
 472 | #![feature = "signal"]
     |    ------------------ the item is gated behind the `signal` feature
...
1061 | pub fn kill<T: Into<Option<Signal>>>(pid: Pid, signal: T) -> Result<()> {
     |        ^^^^

Some errors have detailed explanations: E0425, E0432.
For more information about an error, try `rustc --explain E0425`.
error: could not compile `ttr` (bin "ttr") due to 2 previous errors
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,6 +10,6 @@ clap = {version = "4.1.6", features = ["derive"]}
 crossterm = "0.26.0"
 ctrlc = "3.4.5"
 dirs = "4.0.0"
-nix = "0.29.0"
+nix = {version="0.29.0", features = ["signal"]}
 serde = {version = "1.0.152", features = ["derive"]}
 serde_yaml = "0.9.17"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -5,11 +5,11 @@ version = "0.3.0"
 authors = ["Denis Bazhenov <dotsid@gmail.com>"]
 
 [dependencies]
-anyhow = "1.0.69"
-clap = {version = "4.1.6", features = ["derive"]}
-crossterm = "0.26.0"
-ctrlc = "3.4.5"
+anyhow = "1.0.100"
+clap = {version = "4.5.53", features = ["derive"]}
+crossterm = "0.26.1"
+ctrlc = "3.5.1"
 dirs = "4.0.0"
 nix = {version="0.29.0", features = ["signal"]}
-serde = {version = "1.0.152", features = ["derive"]}
-serde_yaml = "0.9.17"
+serde = {version = "1.0.228", features = ["derive"]}
+serde_yaml = "0.9.34"

